draft-ietf-dnsop-svcb-https.md

```diff
@@ -659,6 +661,25 @@ each RRSet in the Additional section with the same DNSSEC-related records
 that they would send when providing that RRSet as an Answer (e.g. RRSIG, NSEC,
 NSEC3).
 
+## EDNS Client Subnet (ECS) {#ecs}
+
+The EDNS Client Subnet option (ECS, {{!RFC7871}}) allows recursive
+resolvers to request IP addresses that are suitable for a particular client
+IP range.  SVCB records may contain IP addresses (in ipv*hint SvcParams),
+or direct users to a subnet-specific TargetName, so recursive resolvers
+SHOULD include the same ECS option in SVCB queries as in A/AAAA queries.
+
+According to Section 7.3.1 of {{!RFC7871}}, "Any records from \[the
+Additional section\] MUST NOT be tied to a network".  Accordingly,
+resolvers SHOULD treat any records in the Additional section as having
+SCOPE PREFIX-LENGTH zero, and MAY cache them on this basis.  Authoritative
+servers MUST omit such records if they are not suitable for global use.
```

Comment:
Here's an alternative interpretation of "MUST NOT be tied to a network".  This might allow more use of Additional records, resulting in lower latency.

@puneetsood FYI

```suggestion
SOURCE PREFIX-LENGTH zero and SCOPE PREFIX-LENGTH as specified
in the ECS option, and MAY cache them on this basis.  Authoritative
servers MUST omit such records if they are not suitable
for use by any stub resolvers that set SOURCE PREFIX-LENGTH to zero.
```

Reply:
On reflection, I'm leaning toward this interpretation, because it makes it more likely that we can revise RFC 7871 in the future to make these records ECS-enabled without creating a compatibility problem.